apis/Google.Cloud.Spanner.Data/docs/connection_string.md

```diff
@@ -0,0 +1,66 @@
+# Connection string options
+
+Spanner connection strings support the following options:
+
+# Data Source
+
+The project, instance and optionally database to use. The format is
+`projects/{project}/instances/{instance}/databases/{database}`, with
+the `databases/{database}` part being optional.
+
+Examples:
+
+- Data Source=projects/my_project/instances/my_instance
+- Data Source=projects/my_project/instances/my_instance/databases/my_database
+
+# CredentialFile
+
+A path to a JSON service account credential file to use. If this is
+not specified, the application default credentials will be used.
+(These are provided automatically when running on Google Cloud
+Platform, or a JSON service account credential file can be specified
+with the `GOOGLE_APPLICATION_CREDENTIALS` environment variable.)
+
+Example:
+
+- CredentialFile=/path/to/credentials.json
+
+# EnableGetSchemaTable
+
+When set to true (and when targeting .NET 4.5 or .NET Standard 2.0),
```

Comment:
For clarity it might be helpful to add in the brackets: ", not supported in .NET Standard 1.x"

Reply:
Good plan.